### PR TITLE
#76 newrelic upgrade agent

### PR DESCRIPTION
**Pull request by abdul1234e**
﻿## Summary
Addresses Issue # 76

Please note if fully resolves the issue per the acceptance criteria: Y/N
yes
*Describe the pull request here, including any supplemental information needed to understand it.*
Upgrade newrelic agent to latest. This is second attempt to address this issue. In previous attempt i manually updated package-lock.json. hence this is fixed by running npm update newrelic with their dependencies update. 
## Impacted Areas of the Site
- 
none
## Optional Screenshots
none
## This pull request changes...
- [x] expected change 1 : Newrelic agent upgrade to 5.10.0

## This pull request is ready to merge when...
- [x] Feature branch is starts with the issue number
- [] Is connected to its original issue via zenhub 👇
- [x] Tests have been updated 
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [] Server actions captured by logs (manual)
- [] Documentation / readme.md updated (manual)
- [] API docs updated if need (manual)
- [] JSDocs updated (manual)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/package-lock.json [hunk 1]
```diff
@@ -8830,6 +8838,7 @@
       "version": "2.11.1",
       "resolved": "https://registry.npmjs.org/nan/-/nan-2.11.1.tgz",
       "integrity": "sha512-iji6k87OSXa0CcrLl9z+ZiYSuR2o+c0bGuNmXdrhTQTakxytAFsC56SArGYoiHlJlFoHSnvmhpceZJaXkVuOtA==",
+      "dev": true,
```
Comment (briandavidson) [inline frontend/package-lock.json]: @abdul1234e what is this change doing?

Comment (abdul1234e) [reply frontend/package-lock.json]: its updating the dependencies with npm update

**Comment by abdul1234e** [comment]
@briandavidson @mtlaney please review and approve it. Merge conflict got resolved.

**Comment by abdul1234e** [comment]
> It looks like there are still some required steps that are missing for documentation and connecting the PR 
@briandavidson The issue is on other board. Its not letting me connect.

**Pull request merged**